bin/install-prereqs.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Install linters
+go get -u github.com/alecthomas/gometalinter
+gometalinter --install --vendored-linters
+
+# Install buildifier BUILD file validator
+go get -u github.com/bazelbuild/buildifier/buildifier
```

Comment:
yeah let's add it when it actually gets used to keep each commit self-contained.